nslsii/loggers.py

```diff
@@ -0,0 +1,52 @@
+import logging
+import requests
+
+
+class ElasticHandler(logging.Handler):
+    '''
+    Logging handler that submits data to Elastic
+
+    Parameters
+    ----------
+    url: string
+        url to Elastic server
+
+    level: Logging Levels, optional
+        Handler logging level
+    '''
+    def __init__(self, url, level=logging.INFO):
+        self.url = url
+        self.level = level
+        self.response = None
```

Comment:
This is superfluous now.

Reply:
I think url and level are still useful, right?

Reply:
Yes.